app/templates/error/404.html

```diff
@@ -1,13 +1,19 @@
 {% extends "document_download_template.html" %}
-{% block per_page_title %}Page could not be found{% endblock %}
+{% block per_page_title %}Page not found{% endblock %}
 {% block main_content %}
     <div class="grid-row">
       <div class="column-two-thirds">
         <h1 class="heading-large">
-          Page could not be found
+          Page not found
         </h1>
         <p>
-          Check you've entered the correct web address.
+          If you typed the web address, check it is correct.
+        </p>
+        <p>
+          If you pasted the web address, check you copied the entire address.
+        </p>
+        <p>
+          If the web address is correct or you selected a link in an email, contact the team that emailed you.
```

Comment:
Alternatives: 

- `If the web address is correct or you selected a link in an email, contact the sender.`
- `If you selected a link in an email, contact the sender.`